dotnetv3/cross-service/S3ObjectLambdaFunction/README.md

```diff
@@ -0,0 +1,204 @@
+# Transforming data for your application with S3 Object Lambda
+
+
+## Overview
+
+In this tutorial, you learn how to add custom code to standard S3 GET requests to modify the requested object retrieved from S3 so that the object suit the needs of the requesting client or application. Specifically, you learn how to transform all the text in the original object stored in S3 to uppercase through S3 Object Lambda.
```

Comment:
```suggestion
In this tutorial, you learn how to add custom code to standard S3 GET requests to modify the requested object retrieved from S3 so that the object suits the needs of the requesting client or application. Specifically, you learn how to transform all the text in the original object stored in S3 to uppercase through S3 Object Lambda.
```

Reply:
Updated